src/VrpSolver.jl

```diff
@@ -2169,6 +2129,28 @@ function register_solutions(optimizer::VrpOptimizer, bapcodsol)
     end
     optimizer.unmapped_vars_in_sol = unmapped_vars_in_sol
 
+    # retrieving subproblem sols
+    status = c_next(bapcodsol) # skip master solution
+    while status == 1
+        subproblem_id = Int(c_getProblemFirstId(bapcodsol))
+        mult = c_getMultiplicity(bapcodsol)
+        graph = optimizer.user_model.graphs[subproblem_id + 1]
+        arcs_ids = [
```

Comment:
This Vector did not need to be allocated. The number of arcs may be long. I suggest merging it with line 2145.

Reply:
we need a vector of arc ids because of the call to `get_path_uservar_map` in line 2144